release/images.sh

```diff
@@ -22,40 +22,79 @@ LOG_DIR="${WORK_DIR}/logs"
 PROCS_PER_IMG=8
 TEMPLATES_PATH=${TEMPLATES_PATH:-"${PWD}/config/images"}
 
-sign_checksum() {
-    local chksum_file=$1
-
-    if [[ -n "${IMG_SIGN_CMD}" ]]; then
-        log "Signing (custom)" "${chksum_file}"
-        "${IMG_SIGN_CMD}" "${chksum_file}" "${chksum_file}.sig"
-    elif [[ -s "${IMG_SIGN_KEY}" ]]; then
-        log "Signing (openssl)" "${chksum_file}"
-        openssl dgst -sha1 -sign "${IMG_SIGN_KEY}" -out "${chksum_file}.sig" "${chksum_file}"
-    else
-        warn "Skipping signing" "Neither custom signing command nor a signing key provided."
+checksum_and_sign() {
+    # ${1} - Where to output the list of files, checksum & signed files -> release dir
+    # ${2} - File path to image file
+    declare -n _output=${1}
+    local file_list=$2
+
+    for file in ${file_list}; do
+        local chksum_file="${file}-${CHKSUM_FILE_SUFFIX}"
+
+        sha512sum "${file}" > "${chksum_file}"
```

Comment:
This generates shasum with full path of file. It can leak unnecessary details. pushd into the directory and then sign the file(s) should prevent that.